### ISSUE TITLE
Multiple Servers Usage Mode: Round-Robin

### ISSUE THREAD

**Issue by alexdelprete** [alexjustesen/speedtest-tracker#341]
Right now I'm observing a random usage of the configured servers. I have only 2 servers and I expected a round-robin mode of usage for the tests, instead it seems random:

![image](https://user-images.githubusercontent.com/7027842/213457041-00a91d7e-5c5e-4593-a5a9-babf88761e6a.png)

**Comment by alexjustesen** [comment]
So right now it's random by design (reference to code below). So you're looking for test A, next time test B, then A etc.?

https://github.com/alexjustesen/speedtest-tracker/blob/98cf390ac2e6fff8ada7dbb352569d8e12de8143/app/Jobs/SearchForSpeedtests.php#L26

**Comment by alexdelprete** [comment]
Yes, round-robin. If there are 3 servers: 1, 2, 3 and repeat the cycle from 1.

**Comment by alexdelprete** [comment]
No problem, nothing urgent.

fail-over I think should not be mutually exclusive to the rest. I mean, if I select round-robin or random, and a server fails, I would prefer ST to skip to the next one.

**Comment by Bag72** [comment]
My two pennies worth.  Fail over shouldn't be an option.  If there is an issue it should fail safe and perform a test on something else.   The next item on a list or if none then another list of reliable servers.

**Comment by alexdelprete** [comment]
I agree, and it's exactly what I meant in my previous post when I said it shouldn't be mutually exclusive.

**Comment by MartynKeigher** [comment]
+1 on the round-robin approach & the failover! :) 

Also if ServerA is unreachable for any reason, then yes, attempting a speed test against ServerB should take place. eg... 

- Test 1: SevrerA - OK!
- Test 2: SevrerB - OK!
- Test 3: SevrerA - BAD... Changing target to ServerB - OK!
- Test 4: SevrerA - OK!
- Test 5: SevrerB - OK!
... and so on.

**Comment by jackblk** [comment]
Can we also do all of them?

I'm very interested in tracking my speed from my server to multiple locations, there are many use cases for this. Then in the dashboard I can view the result of my locations that I chose. That would be awesome.
